<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C5DishWidget</class>
 <widget class="QWidget" name="C5DishWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>602</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Main</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Main</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="1" colspan="4">
           <widget class="C5LineEditWithSelector" name="lineEdit_3">
            <property name="Field" stdset="0">
             <string>f_comment</string>
            </property>
            <property name="Type" stdset="0">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="4">
           <widget class="C5LineEditWithSelector" name="leName">
            <property name="Field" stdset="0">
             <string>f_name</string>
            </property>
            <property name="Type" stdset="0">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="C5LineEditWithSelector" name="lePart2Name">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="FieldRef" stdset="0">
             <string>f_part2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Type</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="C5LineEditWithSelector" name="lePart2">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="Field" stdset="0">
             <string>f_part</string>
            </property>
            <property name="Type" stdset="0">
             <string>1</string>
            </property>
            <property name="First" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="C5LineEditWithSelector" name="leCode">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="Field" stdset="0">
             <string>f_id</string>
            </property>
            <property name="Primary" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Code</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Description</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QPushButton" name="btnNewType">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../resources/res.qrc">
              <normaloff>:/new.png</normaloff>:/new.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="C5LineEditWithSelector" name="lePart1Name">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="FieldRef" stdset="0">
             <string>f_part1</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Menu</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="C5TableWidget" name="tblPricing">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <column>
             <property name="text">
              <string>Menu record</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Menu code</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Menu name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Price</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Store</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Print1</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Print2</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>X</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Additional</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Queue</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="C5CheckBox" name="checkBox_2">
            <property name="text">
             <string>Append service fee</string>
            </property>
            <property name="Field" stdset="0">
             <string>f_service</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="C5CheckBox" name="checkBox">
            <property name="text">
             <string>Can discount</string>
            </property>
            <property name="Field" stdset="0">
             <string>f_discount</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="C5CheckBox" name="checkBox_3">
            <property name="text">
             <string>Reminder</string>
            </property>
            <property name="Field" stdset="0">
             <string>f_remind</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="C5LineEditWithSelector" name="lineEdit_2">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="Field" stdset="0">
             <string>f_queue</string>
            </property>
            <property name="Type" stdset="0">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="C5LineEditWithSelector" name="leColor">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="Field" stdset="0">
             <string>f_color</string>
            </property>
            <property name="Type" stdset="0">
             <string>1</string>
            </property>
            <property name="SetColor" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Recipe</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Recipes</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="btnAddRecipe">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../resources/res.qrc">
                 <normaloff>:/add.png</normaloff>:/add.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnRemoveRecipe">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../resources/res.qrc">
                 <normaloff>:/remove.png</normaloff>:/remove.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnPrintRecipe">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../resources/res.qrc">
                 <normaloff>:/print.png</normaloff>:/print.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>584</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="C5TableWidget" name="tblRecipe">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>2</verstretch>
             </sizepolicy>
            </property>
            <column>
             <property name="text">
              <string>Record</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Goods code</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Goods name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Qty</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Unit</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Price</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Total</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>473</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Total self cost</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="C5LineEdit" name="leTotal">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>C5TableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">c5tablewidget.h</header>
  </customwidget>
  <customwidget>
   <class>C5LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>C5LineEditWithSelector</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineeditwithselector.h</header>
  </customwidget>
  <customwidget>
   <class>C5CheckBox</class>
   <extends>QCheckBox</extends>
   <header location="global">c5checkbox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
